tests/arch: arm_thread_swap: Increase no_optim case flash requirement

erwango opened the pull request:
On most targets, application flash size for no_optimization test
configuration is slightly higher than 128 Kbytes.
Updating requirement to the next upper flash size.

This fixes issue reported by shippable in https://github.com/zephyrproject-rtos/zephyr/pull/18806

ioannisg reviewed (commented):
This allows to execute the test for boards with exactly 192K, right? E.g. nRF52810_pca10040...

ulfalizer commented:
> How does the check patch pass with the commit title truncated?

The title is literally `tests/arch: arm_thread_swap: Increase flash requirement for no_optim ...` in the commit too, and that's just short enough to not trigger it.

Looks a bit weird though.

ioannisg commented:
> 
> 
> > How does the check patch pass with the commit title truncated?
> 
> The title is literally `tests/arch: arm_thread_swap: Increase flash requirement for no_optim ...` in the commit too, and that's just short enough to not trigger it.
> 
> Looks a bit weird though.

Oh, I thought it was like this due to some truncation :)

erwango commented:
> This allows to execute the test for boards with exactly 192K, right? E.g. nRF52810_pca10040...

@ioannisg, this is `min_flash`, so filters out targets with lower flash size (this is used to work this way at least).

ioannisg commented:
> 
> 
> > This allows to execute the test for boards with exactly 192K, right? E.g. nRF52810_pca10040...
> 
> @ioannisg, this is `min_flash`, so filters out targets with lower flash size (this is used to work this way at least).

Right, I was only unsure if the inequality is strict or not @erwango .
So if the SoCs with exactly 192 are filtered in or out.

erwango commented:
@ioannisg, ok was surprised by the question. Get it better now.

So, I ran the following command:
`./scripts/sanitycheck -T tests/arch/arm/arm_thread_swap/ -p nRF52810_pca10040`

And actually the board is filtered out w/ or w/o the modification the board is filtered out anyway.
Addind the following in board yaml file, the board is tested even when `min_flash: 192` 
```
testing:
  default: true
```

ioannisg commented:
> 
> 
> @ioannisg, ok was surprised by the question. Get it better now.
> 
> So, I ran the following command:
> `./scripts/sanitycheck -T tests/arch/arm/arm_thread_swap/ -p nRF52810_pca10040`
> 
> And actually the board is filtered out w/ or w/o the modification the board is filtered out anyway.
> Adding the following in board yaml file, the board is tested even when `min_flash: 192`
> 
> ```
> testing:
>   default: true
> ```

So testing: default: true is required to include a board in SanityCheck testing?

ioannisg reviewed (changes requested):
@erwango please, reword the commit message title.

erwango committed:
tests/arch: arm_thread_swap: Increase no_optim case flash requirement

On most targets, application flash size for no_optimization test
configuration is slightly higher than 128 Kbytes.
Updating requirement to the next upper flash size.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

erwango commented:
@ioannisg, done
